### ISSUE TITLE
Failures of "gcloud container clusters get-credentials" should be displayed

### ISSUE THREAD

**Issue by volpino** [google/kctf#370]
See https://github.com/google/kctf/blob/44da9297885222d5f22995eda60fb39380a44d0a/dist/bin/kctf-cluster#L85

Errors are sent to /dev/null and there is no way of knowing what happened if that command failed. Errors should be reported to the user.

**Comment by sirdarckcat** [comment]
i agree that the errors are hard to track down, we need a better infra for error display
